### ISSUE TITLE
Open Tabs icon doesn't read correctly the number of items and item in list

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16417]
### Prerequisites
Have three open tabs.
Talkback should be enabled
Grid view in Tabs Tray should be enabled

## Steps to reproduce
1. Open The Tabs Tray
2. Tap the Tabs counter button

### Expected behavior
"Selected, 3open tabs. Tap to switch tabs. Tab 1of 3, in list, 3 items"

### Actual behavior
"Selected, 3 open tabs. Tap to switch tabs. Tab 1of 2, in list, 2 items"

### Device information

* Android device: Google Pixel 3a (Android 11)
* Fenix version: Nightly Build 201106

### Notes
❗ Reproducible with the usual Tabs Tray as well

► Screenshot
![rsz_screenshot_20201106-115115](https://user-images.githubusercontent.com/51314259/98355711-36341e80-202b-11eb-8e90-beef79ab466d.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-8672)

**Comment by mcarare** [comment]
This is the expected behavior. "Tab 1of 2, in list, 2 items" is in reference to the number of tabs on the bottom sheet (normal and private.)

**Comment by AndiAJ** [comment]
From my point of view having **3 open tabs** displayed in the Tabs Tray, they should be counted accordingly

**Comment by mcarare** [comment]
> From my point of view having **3 open tabs** displayed in the Tabs Tray, they should be counted accordingly

The open tabs are counted accordingly, it says: "3 open tabs"

**Comment by Mugurell** [comment]
The difference between 
the expected behavior (what happens when using list view) and
the actual behavior (what happens when using grid view)
is because of what views are focused immediately after the tab tray is opened:
- when using list view the current selected browser tab is focused
- when using grid view the current selected bottom sheet tab (normal or private tabs icon) is focused

So a quick fix here (until we get UX feedback in https://github.com/mozilla-mobile/fenix/issues/16415 for how the bottom sheet tabs should be read to eliminate confusion), is to just make sure the grid tab is announced when opening the tabs tray.

**Comment by mcarare** [comment]
Waiting for UX feedback in UX feedback in #16415.

**Comment by boek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1812892

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
